src/package.lisp

```diff
@@ -60,9 +60,17 @@
    #:ed25519-key-x #:ed25519-key-y
 
    ;; pseudo-random number generators
-   #:pseudo-random-number-generator #:list-all-prngs #:make-prng #:random-data
```

Comment:
It was a tough call to remove `#:PSEUDO-RANDOM-NUMBER-GENERATOR`, since it was exported and hence part of the external interface. However, it was only exported to allow user-defined PRNG classes, but the methods necessary to actually define PRNG classes were never exported, and thus any code which _did_ extend `PSEUDO-RANDOM-NUMBER-GENERATOR` was relying on internals.